core/contracts/financial_templates/PricelessPositionManager.sol

```diff
@@ -93,7 +94,12 @@ contract PricelessPositionManager is Testable {
         Token mintableToken = new Token(_syntheticName, _syntheticSymbol, 18);
         tokenCurrency = Token(address(mintableToken));
         finder = Finder(_finderAddress);
-        priceIdentifer = _priceFeedIdentifier;
+        priceIdentifer = _priceIdentifier;
+
+        IdentifierWhitelistInterface identiferWhiteList = IdentifierWhitelistInterface(
```

Comment:
Can we make the method return type `IdentifierWhitelistInterface` rather than doing the cast here?

Reply:
This refactor should be done in a separate PR for methods `_getStoreAddress` and `_getOracleAddress`. I created an issue to do this #913